pytealext/assemble.py

```diff
@@ -3,22 +3,20 @@
 from pyteal import CompileOptions, Expr, TealBlock, TealOp, TealSimpleBlock
 
 
-def assemble_steps(steps: Iterable[TealOp or Expr], options: CompileOptions) -> tuple[TealBlock, TealSimpleBlock]:
+def assemble_steps(steps: Iterable[TealOp | Expr], options: CompileOptions) -> tuple[TealBlock, TealSimpleBlock]:
```

Comment:
if we are not using Union, then support for python3.9 should be dropped

Reply:
the reqs are already python 3.10+ 👌

Reply:
ok so in `setup.cfg` we should change `python_requires` to `>=3.10`

Reply:
already done ^